ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolSpecBuilder.java

```diff
@@ -284,6 +294,8 @@
 
     final BlockHeaderValidator<T> blockHeaderValidator =
         blockHeaderValidatorBuilder.apply(difficultyCalculator);
+    additionalBlockHeaderValidationRules.forEach(blockHeaderValidator::addRule);
```

Comment:
This will require a tiny bit of replumbing.  We can have `blockHeaderValidatorBuilder` return the builder instead of the fully assembled `blockHeaderValidator`. Then we call `addRule` on the builder, then build and store.

Reply:
Done